demo/dynamic_image.cpp

```diff
@@ -0,0 +1,74 @@
+/*
+    Copyright 2019 WerWolv
```

Comment:
are you sure of the copyright ?

Reply:
Never worked with PRs and licensing, should it just be `Copyright 2020-2021 natinusala`?

Reply:
No, the copyright is for the person who wrote the file so that would be 2021 [your name]!